### PR TITLE
Unify more Oak Functions crates into mods

### PR DESCRIPTION
**Pull request by tiziano88**

### CODE REVIEW

File: oak_functions_service/src/logger.rs [hunk 1]
```diff
@@ -14,8 +14,6 @@
 // limitations under the License.
 //
 
-#![no_std]
-
```
Comment (jul-sh) [inline oak_functions_service/src/logger.rs:18]: slightly unfortunate consequence that these are now no longer no std compatible

**Review by jul-sh — APPROVED** [review]
thanks, lgtm

File: oak_functions_service/src/logger.rs (hunk 1 above)
Comment (tiziano88) [reply oak_functions_service/src/logger.rs:18]: Quite the opposite, now everything in the crate is by necessity no_std compatible

Comment (jul-sh) [reply oak_functions_service/src/logger.rs:18]: oh nice

**Pull request merged**